lib/util/git/index.ts

```diff
@@ -42,6 +43,50 @@ export { setPrivateKey } from './private-key';
 // TODO: fix upstream types https://github.com/steveukx/git-js/issues/704
 const ResetMode = (simpleGit.default as any).ResetMode as typeof _ResetMode;
 
+// Retry parameters
+const retryCount = 5;
+const delaySeconds = 3;
+const delayFactor = 2;
+
+// A generic wrapper for simpleGit.* calls to make them more fault-tolerant
+export async function gitRetry<T>(gitFunc: () => Promise<T>): Promise<any> {
```

Comment:
@viceice left a comment about this earlier, but I don't think it was fully addressed.

```suggestion
export async function gitRetry<T>(gitFunc: () => Promise<T>): Promise<T> {
```